packages/itwin/grouping-mapping-widget/src/test/GroupingMappingCustomUI.test.tsx

```diff
@@ -79,6 +80,11 @@ jest.mock("../components/context/MappingClientContext", () => ({
   useMappingClient: () => mappingClientMock.object,
 }));
 
+jest.mock("../components/context/GroupsClientContext", () => ({
+  ...jest.requireActual("../components/context/GroupsClientContext"),
+  useGroupsClient: ()=> groupsClientMock.object,
```

Comment:
Formatting looks off.

Reply:
Reformatted to follow the docs.

Reply:
The curly brace and parenthesis were fine. There is a missing space between `()▮=> groupsClientMock.object`

Reply:
Space added.